xblock/core.py

```diff
@@ -174,6 +174,15 @@ def validate(self):
         """
         return Validation(self.scope_ids.usage_id)
 
+    def ugettext(self, text):
```

Comment:
Why are we adding this particular shortcut to the base XBlock? Just so that we can call `self.ugettext(...)` easily, rather than needing to explicitly say `self.runtime.service(self, 'i18n').ugettext(...)`?